src/Sylius/Bundle/AdminBundle/Form/Extension/ProductTypeExtension.php

```diff
@@ -14,16 +14,23 @@
 namespace Sylius\Bundle\AdminBundle\Form\Extension;
 
 use Sylius\Bundle\CoreBundle\Form\Type\Product\ProductImageType;
+use Sylius\Bundle\ProductBundle\Form\Type\ProductAssociationsType;
 use Sylius\Bundle\ProductBundle\Form\Type\ProductType;
 use Symfony\Component\Form\AbstractTypeExtension;
 use Symfony\Component\Form\FormBuilderInterface;
 use Symfony\UX\LiveComponent\Form\Type\LiveCollectionType;
 
 final class ProductTypeExtension extends AbstractTypeExtension
 {
+    /**
+     * @param array<string, mixed> $options
+     */
     public function buildForm(FormBuilderInterface $builder, array $options): void
     {
         $builder
+            ->add('associations', ProductAssociationsType::class, [
+                'label' => false,
+            ])
```

Comment:
It shouldn't be needed here as this field exists in original ProductType from the ProductBundle